src/main/java/com/epam/test_generator/pojo/JiraStory.java

```diff
@@ -30,6 +31,10 @@
 
 
     public JiraStory(Issue issue) {
+        if(issue.getIssueType().isSubtask()) {
+            throw new JiraRuntimeException("Cannot create JiraStory from substory [key:" + issue.getKey() + "]");
```

Comment:
Throwing an exception is pretty slow operation, if you don't make null Throwable.
Kindly, think about it and check one more time, if stack trace of an exception is really needed somewhere.
If so, then Ok.
If no, make it in way like this:
`throw new JiraRuntimeException("Cannot create JiraStory from substory [key:" + issue.getKey() + "]", null);`